bitnami/keycloak/values.yaml

```diff
@@ -92,6 +92,36 @@ auth:
     ## Password to access the truststore when it's password-protected.
     ##
     truststorePassword: ""
+    ## Init container parameters:
```

Comment:
Please add these new parameters to the README.

Reply:
This has been completed.